Sources/Views/Collections/CollectionSection.swift

```diff
@@ -377,10 +376,13 @@ private func sectionNavigation(
         .class([
           Class.flex.items.center,
           Class.grid.center(.mobile),
+          Class.padding([
+            .desktop: [.leftRight: 5],
+            .mobile: [.leftRight: 3],
+          ])
         ]),
         .style(
-          height(.px(88))
```

Comment:
I removed the explicit height on the bottom navigation and instead rely on the padding to define its height. this allows us to make the border extend from the top to the bottom of the row